app/main.py

```diff
@@ -17,3 +17,7 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+sorted_variables = {}
+sorted_variables["mutable"] = my_favourite_films, marks, collection_of_coins
+sorted_variables["immutable"] = lucky_number, pi, one_is_a_prime_number, name
+sorted_variables["immutable"] += profile_info,
```

Comment:
make it without double sorted_variables["immutable"]

Reply:
changes is ready